src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/OrganizationMapperTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Organization;
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class OrganizationMapperTest {
+
+    @Test
+    void testMapPerformingOrganization() {
+        final Message message = new Message(new ArrayList<>());
```

Comment:
Because we are calling this line of code in each test in this class, do you think that beforeEach might be a good idea?

Reply:
It's ok for now. This test will change once the story to develop the organization mapper is picked up